### ISSUE TITLE
Training script

### ISSUE THREAD

**Issue by Altimis** [nikicc/twitter-emotion-recognition#16]
Thank you so moch for these briliant models. Can you please provide us with the code that you used in otder to train the models ?

**Comment by nikicc** [comment]
Hi @Altimis. It's nice to hear they are useful.

About the code, I don't really have that at hand and in a format, I could easily share. Sorry about that 🙈 . However, you can see the architecture of the classifier in the models that are loaded. To train them, you can use any gradient-based method, I believe I used RMSProp.

**Comment by Altimis** [comment]
Thank you @nikicc, I'll try to analyse the model's architecture. How much data did you use tho ?

**Comment by nikicc** [comment]
@Altimis training data had about 320,000 examples for Ekman, 480,000 for Plutchik, and 3,900,000 for POMS.

**Comment by Altimis** [comment]
Oh okey, the labeling must be hard, unless there are some opensource datasets for this kind of tasks

**Comment by nikicc** [comment]
I didn't find any opensource datasets, at least not of that size.

For labelling, I used distant supervision: i.e. we search for tweets containing hashtags, used them as the target variable, removed them from the tweet content, and then train classifiers. If you're interested more into detail, check the paper — there is a link at the bottom of the [README.md](https://github.com/nikicc/twitter-emotion-recognition/blob/master/README.md) file.

**Comment by Altimis** [comment]
@nikicc  interesting idea ! I'll check your paper for sure. Thank you again for your work.
